<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitRibbon</class>
 <widget class="QWidget" name="UnitRibbon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="RibbonButton">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>0</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="plantNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#55aa00;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="plantTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfigureButton">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/ConfigureButtonIcon.PNG</normaloff>:/ConfigureButtonIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="waterLevelWarning">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLevelWarning">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>plantNameLabel</zorder>
  <zorder>plantTypeLabel</zorder>
  <zorder>waterLevelWarning</zorder>
  <zorder>batteryLevelWarning</zorder>
  <zorder>ConfigureButton</zorder>
  <zorder>RibbonButton</zorder>
 </widget>
 <resources>
  <include location="Resources/biobloom.qrc"/>
 </resources>
 <connections/>
</ui>
